juju/client/connection.py

```diff
@@ -326,6 +338,12 @@ def current_controller(self):
         output = yaml.safe_load(output)
         return output.get('current-controller', '')
 
+    def current_model(self, controller_name=None):
+        models = self.models()[controller_name or self.current_controller()]
```

Comment:
I think this would be better as two steps vs combined like this. It's kind of unclear what's going on at first glance.